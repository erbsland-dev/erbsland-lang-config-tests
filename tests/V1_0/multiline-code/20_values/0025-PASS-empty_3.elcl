# Erbsland Configuration Language Test File
[multi_line_code]
value 03: ```
    ``` # Comment 3
