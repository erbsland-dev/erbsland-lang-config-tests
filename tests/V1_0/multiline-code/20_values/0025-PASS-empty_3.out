multi_line_code = SectionWithNames()
multi_line_code.value_03 = Text("")
